tests/conftest.py

```diff
@@ -1,8 +1,16 @@
 import numpy as np
 import pytest
+import tensorflow as tf
+from packaging.version import Version
 
 from gpflow.kernels import SquaredExponential
 
+# TODO: It would be great to make serialisation work in general.
```

Comment:
Do we know what is preventing this in more technical detail? Would it possible to post this as an issue in either GPflux or GPflow?

Reply:
> Do we know what is preventing this in more technical detail? Would it possible to post this as an issue in either GPflux or GPflow?

Dunno. I tracked the problem down to serialisation of GPflow Parameters, and spent ~1hour trying to get that to work, but then decided it wasn't that important and moved on. 

We already have a ticket to enable serialisation of GPflow in general:
https://github.com/GPflow/GPflow/issues/1658